### PR TITLE
Django views hw

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: app/concierge/concierge/settings.py [hunk 1]
```diff
@@ -53,11 +54,13 @@
 ]
 
 ROOT_URLCONF = 'concierge.urls'
+print(BASE_DIR)
```
Comment (dimigor) [inline app/concierge/concierge/settings.py]: remove print

File: app/concierge/mycore/urls.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.urls import path, include
+
+from .views import TenantsView, TenantView, RoomsView, RoomView, RoomFormView, TenantFormView, JournalView, JournalFormView
```
Comment (dimigor) [inline app/concierge/mycore/urls.py]: will be better if you will not use `.` in import

File: app/concierge/mycore/views.py [hunk 3]
```diff
@@ -1,3 +1,97 @@
...
+    template_name = 'tenant.html'
+
+    def get_tenant(self, tenant_id):
+        return self.data[tenant_id]
+
+
+class TenantFormView(FormView):
+    template_name = 'forms/tenant_form.html'
+    form_class = TenantForm
+    success_url = '/core/tenants'
```
Comment (dimigor) [inline app/concierge/mycore/views.py]: use django function `reverse()`

File: app/concierge/templates/forms/tenant_form.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    <div>
+        <label for="date_of_birth">Date of birth: </label>
+        <input type="date" id="date_of_birth" name="date_of_birth">
+    </div>
+    <div>
+        <label for="phone">Phone: </label>
+        <input type="text" id="phone" name="phone">
+    </div>
+    <input type="submit"/>
+</form>
```
Comment (dimigor) [inline app/concierge/templates/forms/tenant_form.html]: endline

Comment (sudakevych-andrii) [reply app/concierge/templates/forms/tenant_form.html]: Done

File: app/concierge/concierge/settings.py (hunk 1 above)
Comment (sudakevych-andrii) [reply app/concierge/concierge/settings.py]: Done

**Pull request merged**